docusaurus.config.js

```diff
@@ -5,20 +5,10 @@ module.exports = {
   tagline: 'Invincible applications, invisible infrastructure',
   url: 'https://docs.temporal.io',
   baseUrl: '/',
+  onBrokenLinks: 'warn',
   favicon: 'img/favicon.svg',
   organizationName: 'temporalio', // Usually your GitHub org/user name.
   projectName: 'temporal-documentation-legacy-v2', // Usually your repo name.
-  plugins: [
-    '@docusaurus/plugin-google-gtag',
-    [
-      '@docusaurus/plugin-sitemap',
```

Comment:
Why is sitemap removed?

Reply:
it is already part of @docusaurus/preset-classic according to https://v2.docusaurus.io/docs/presets/#docusauruspreset-classic
Starting the recent versions of docusaurus, it has started warning at build time that this plugin is set up twice so i removed the second declaration. 

To address the comment, i've pushed a new commit that adds explicit settings for sitemap plugin + comments for clarity https://github.com/temporalio/documentation/pull/80/commits/852d5952b2ea1ad059577bb818b3bf22d86f5c68